galata/package.json

```
@@ -37,13 +37,19 @@
     "start": "jupyter lab --config ./jupyter_server_test_config.py",
     "start:detached": "(npm run start&)",
     "start:doc": "jupyter lab --config ./jupyter_server_test_config.py --extensions-in-dev-mode",
+    "start:rtc": "jupyter lab --config ./jupyter_server_test_config.py --collaborative --LabApp.collaborative_document_save_delay=0",
     "test": "playwright test",
+    "test:base": "playwright test --project galata jupyterlab",
+    "test:base:update": "playwright test --project galata jupyterlab --update-snapshots",
     "test:benchmark": "jlpm run test -c playwright-benchmark.config.js",
     "test:benchmark:update": "BENCHMARK_NUMBER_SAMPLES=1 jlpm run test -c playwright-benchmark.config.js --update-snapshots",
     "test:debug": "PWDEBUG=1 playwright test",
-    "test:doc": "jlpm run test -c playwright-documentation.config.js",
-    "test:doc:update": "jlpm run test -c playwright-documentation.config.js --update-snapshots",
+    "test:doc": "jlpm run test --project documentation",
+    "test:doc:update": "jlpm run test --project documentation --update-snapshots",
     "test:report": "playwright show-report",
+    "test:rtc": "playwright test --project rtc --workers 1",
```

Could you please move that option to the `playwright.config.js` file? See [doc](https://playwright.dev/docs/test-configuration#testing-options)

```suggestion
    "test:rtc": "playwright test --project rtc",
```

---

I tried to add it to the config file before adding it to the command, but it didn't work.
I only want to prevent parallelize in the rtc tests, so I tried adding it here
https://github.com/jupyterlab/jupyterlab/blob/7ec8d1130b8413d96bdceb55090f66cc9eb7e6e5/galata/playwright.config.js#L33-L37
but it doesn't work.

---

Thanks indeed that flag is not available on a per project basis. For reference here is the appropriate [documentation](https://playwright.dev/docs/api/class-testproject#test-project-use)